docs/SPIDER_TYPE.md

```diff
@@ -0,0 +1,23 @@
+## Types of spiders
+
+The vast majority of spiders are collecting first party data about a specific brand/chain.
+We sometimes call these "brand spiders", these should be well categorised and branded by us, and have original data from
```

Comment:
Can examples be provided, as well as what should be excluded? Obviously this includes retail chains and restaurant chains. But what about government-operated assets that are POIs people are usually interested in such as public playgrounds, public toilets, etc? And what about branded bike rental locations, car share locations and electric car charging stations (which may be colocated with another feature such as a hotel or shopping centre)? At what point does something stop being a POI and becomes an `infrastructure_spider` -- park benches, rubbish bins, drinking fountains, etc?